test/augmentation/test_augmentation.py

```diff
@@ -195,7 +195,7 @@ def _test_random_p_0_implementation(self, params):
         augmentation = self._create_augmentation_from_params(**params, p=0.0)
         test_input = torch.rand((2, 3, 4, 5), device=self.device, dtype=self.dtype)
         output = augmentation(test_input)
-        assert (output == test_input).all()
+        assert_close(output, test_input)
```

Comment:
@gau-nernst Actually the ideal is to use `self.assert_closed for tests that inherit from BaseTester
https://github.com/kornia/kornia/blob/master/kornia/testing/__init__.py#L124

Reply:
Oh I didn't know that exist. I can make another PR to change all existing `assert_close()` to `self.assert_close()`.

One question. Why don't we merge the logic of `self.assert_close()` into `assert_close()`? Perhaps there is a reason to have a wrapper around `assert_close()` (which is a wrapper around `torch.testing.assert_close()`?

Reply:
because of this https://github.com/kornia/kornia/blob/23f14118538c9660332ea82f74a93cfd4f6c206d/kornia/testing/__init__.py#L275-L278 /cc @johnnv1